cmd/ocsp-responder/main.go

```diff
@@ -163,7 +165,16 @@ func main() {
 
 		if url.Scheme == "mysql+tcp" {
 			auditlogger.Info(fmt.Sprintf("Loading OCSP Database for CA Cert: %s", c.Common.IssuerCert))
-			source, err = makeDBSource(config.Source, c.Common.IssuerCert, c.SQL.SQLDebug)
+			dbMap, err := sa.NewDbMap(config.Source)
+			cmd.FailOnError(err, "Could not connect to database")
+			if c.SQL.SQLDebug {
+				sa.SetSQLDebug(dbMap, true)
+			}
```

Comment:
Maybe this is related to the test functions, which don't do `sa.SetSQLDebug`?  Why don't they do that?